### DISCUSSION TITLE
Welcome to Folium Discussions!

### DISCUSSION THREAD

**Discussion by Conengmo** [python-visualization/folium#1951] [Announcements]
## 👋 Welcome!

We’re using Discussions as a place to connect with other members of our community. We'll start by using it for maintainer team communications. But it's open for anyone to post things that are related to Folium, but are not a bug report or feature suggestion.

We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hansthen** [comment]
Hi I am Hans. I live in the Netherlands. Father of three. Hobbies: chess, reading, weightlifting, but most of all spending time with family.

I use Folium mostly through `streamlit-folium` at work. I really like the Folium package, as it allows me to render maps from Python. It is a big upgrade from having to do part of the work in Python and then another in Leaflet and Javascript.

**Comment by ocefpaf** [comment]
Hi I'm Filipe. I live in Brazil but I'm in Sweden for this year. Father of one. Hobbies: retro games, hiking, and swimming in the ocean.

I became a Folium maintainer after the author left for another job. At the time we were using it a lot for my day job and we could not let Folium disappear. While there are many alternatives to it now, Folium is still the easier one to create maps and hack into the HTML/JS.

I've been trying to find new maintainers for Folium for a while and I could not be luckier with Frank and Hans! They are doing a much better job than I ever could.
